test/unit/org/apache/cassandra/cql3/validation/operations/AlterTest.java

```diff
@@ -702,26 +702,15 @@ public void testAlterTableWithCompression() throws Throwable
         alterTable("ALTER TABLE %s WITH compression = { 'enabled' : 'false'};");
         assertSchemaOption("compression", map("enabled", "false"));
 
-        assertAlterTableThrowsException(ConfigurationException.class,
-                                        "Missing sub-option 'class' for the 'compression' option.",
-                                        "ALTER TABLE %s WITH  compression = {'chunk_length_in_kb' : 32};");
```

Comment:
As I can see the option `chunk_length_in_kb` should remain, right? We are removing `chunk_length_kb`.

Reply:
I was thinking we do not need the whole statement but I need to get it back. We need to read this in the context of what it does. It was testing what happens if there is not `class` in `compression`. In that case it should fail as it is missing. I probably removed that by accident when removing stuff for sstable_compression option.

The solution here is to not remove this test.